akka-docs/src/main/paradox/stream/operators/Source-or-Flow/groupedWeighted.md

```diff
@@ -0,0 +1,40 @@
+# groupedWeighted
+
+Accumulate incoming events until the combined weight of elements is greater than or equal to the minimum weight and then pass the collection of elements downstream.
+
+@ref[Simple operators](../index.md#simple-operators)
+
+## Signature
+
+@apidoc[Source.groupedWeighted](Source) { scala="#groupedWeighted(minWeight:Long)(costFn:Out=&gt;Long):FlowOps.this.Repr[scala.collection.immutable.Seq[Out]]" java="#groupedWeighted(long,akka.japi.function.Function)" }
+@apidoc[Flow.groupedWeighted](Flow) { scala="#groupedWeighted(minWeight:Long)(costFn:Out=&gt;Long):FlowOps.this.Repr[scala.collection.immutable.Seq[Out]]" java="#groupedWeighted(long,akka.japi.function.Function)" }
+
+
+## Description
+
+Chunk up this stream into groups of elements that have a cumulative weight greater than or equal to the `minWeight`, with the last group possibly smaller than requested `minWeight` due to end-of-stream.
+
```

Comment:
It would make sense to list the related operators here: grouped, groupedWeightedWithin (and from those) (as the "see also" [here](https://doc.akka.io/docs/akka/snapshot/stream/operators/Source-or-Flow/ask.html#description))

Reply:
I added references for all four variants of `grouped`. Let me know if my changes work. Thanks.